Sources/AppcuesKit/Presentation/Debugger/FloatingView/DebugView.swift

```diff
@@ -124,8 +127,9 @@ internal class DebugView: UIView, FloatingViewDelegate {
 
     // MARK: Overrides
 
-    override init(frame: CGRect) {
-        super.init(frame: frame)
+    init(mode: DebugMode) {
```

Comment:
I might try and make it where `DebugView` is entirely agnostic to `DebugMode`. Maybe have the `DebugViewController` be the `FloatingViewDelegate` instead of this (that's my bad from how it was), and then the VC could also toggle the icon on the floating view?

Seems less than ideal to handle interactions and state in the view like this (again, my bad). It's also

Reply:
This worked out nicely I think, simpler - thanks